prow/plank/plank_test.go

```diff
@@ -237,6 +238,7 @@ func TestSyncJenkinsJob(t *testing.T) {
 					CompletionTime: time.Now(),
 				},
 			},
+			pendingJobs:      map[string][]kube.ProwJob{},
```

Comment:
Rather than lots of empty maps, I would suggest making it handle `nil` and just using the default.